book/contributions/jh_cbm_nfdi4chem_Repo/nfdi4chem_repo.md

```diff
@@ -0,0 +1,72 @@
+Authors: [Johannes Hunold](https://orcid.org/0000-0002-4378-6061), [Christian Bonatto Minella](https://orcid.org/0000-0001-7696-7662)
+
+# About NFDI4Chem
+
+[NFDI4Chem](https://www.nfdi4chem.de/) is the Chemistry Consortium in the National Research Data Infrastructure for Germany [NFDI](https://www.nfdi.de/?lang=en) 

```

Comment:
This is repeated in the other NFDI4 document. Happy to have the advertisement in both but think its worth discussing.